packages/bot/src/lib/api/surveyPayload.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const endpoint = 'new-member-survey';
+
+module.exports = ({ request }) => ({
+  async getPayload() {
```

Comment:
Do we prefer just `get` as it will already be prefixed by surveyPayload.
```suggestion
  async get() {
```